src/COVID19/simulation.py

```diff
@@ -165,15 +171,15 @@ def simulations(self, n_simulations):
                 if self.verbose:
                     print("Current timestep:", self.timestep)
                 
-                current_state = self.env.step(current_action)
-                next_action = self.agent.step(current_state)
+                next_state = self.env.step(self.current_action)
```

Comment:
**NB**: this code was actually broken before, so now this `simulation()` function works.